.github/workflows/autodocs-images.yaml

```diff
@@ -79,7 +92,6 @@ jobs:
           commit-message: Update Images Reference
           title: "[AutoDocs] Update Images Reference Docs"
           body: "${{ env.CHANGELOG }}"
-          signoff: true
```

Comment:
Does the signing happen implicitly with GitHub's key, or should this stay in to keep the Enforce integration happy?

Reply:
This was breaking the workflow: https://github.com/erikaheidi/academy/actions/runs/5580884160

The error said:
```

main
Unexpected error: error: gpg failed to sign the data:
[GNUPG:] BEGIN_SIGNING
failed to create rekor client: unable to fetch rekor public key from rekor: Get "https://rekor.sigstore.dev/api/v1/log/publicKey": GET https://rekor.sigstore.dev/api/v1/log/publicKey giving up after 1 attempt(s): context deadline exceeded
Error: failed to create rekor client: unable to fetch rekor public key from rekor: Get "https://rekor.sigstore.dev/api/v1/log/publicKey": GET https://rekor.sigstore.dev/api/v1/log/publicKey giving up after 1 attempt(s): context deadline exceeded
```
I can have another look and try to figure it out, I think something has probably changed with gitsign recently? Do you know @jamonation ?

Reply:
I think this was an intermittent issue, I just run it again in my forked repo and it worked.